modules/naplicense/src/signingscheme.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace nap
+{
+	/**
+	 * Signing schemes. Used by the nap::LicenseService to validate a license.  
+	 * Signing scheme used to generate a license with the 'licensegenerator' tool must
+	 * match the signing scheme passed to the nap::LicenseService.
+	 */
+	enum class NAPAPI SigningScheme
```

Comment:
@cklosters should this be in a new namespace instead of directly under `nap`? I'm in the middle of taking a swipe at the RTTI topic.

Reply:
You could introduce a new namespace, in this case 'license'. We often do this for types associated with a specific module to improve readability and search, for example: `nap::uniform::modelMatrix` or `nap::icon::help` (coming in new release). But in this case, because it's a one-off and the name is rather unique already, I don't see the need to do that right now. `ESigningScheme` is fine.